docs/qcfractal/source/index.rst

```diff
@@ -18,18 +18,32 @@ QCFractal emphasizes the following virtues:
 - **Elasticity:** Scale from a single laptop to a dozen simultaneous supercomputers.
 - **Transferability:** Run many different quantum chemistry, semiempirical, or force field programs with a single unified interface.
 
+QCFratal within the QCArchive stack
+-----------------------------------
+
+Within the QCArchive stack, QCFractal is responsible for:
+
+- Accepting requests for calculations, submitted from a client such as `QCPortal <http://docs.qcarchive.molssi.org/projects/qcportal/en/latest/>`_. 
+- Running these calculations on disparate compute resources through Quantum Chemistry programs and procedures supported by `QCEngine <http://docs.qcarchive.molssi.org/projects/qcengine/en/latest/>`_. 
+- Storing and organizing the results of these calculations in a database.
+- Providing access to stored results to clients such as QCPortal.
+
+The diagram below illustrates a representative data flow:
+
+.. image:: media/QCEcosystem.png
+   :width: 800px
+   :alt: QCArchive ecosystem
+   :align: center
```

Comment:
The image likely needs an update. The party parrot as mentioned should probably be replaced, QCPortal needs a font similar to the other programs, and "compute through engine" reads awkwardly in the image. Did you obtain the source image from @Lnaden?

Reply:
@dgasmith thanks for the feedback!